### ISSUE TITLE
Missing ref information on hiking routes with osmc symbol

### ISSUE THREAD

**Issue by Janjko** [waymarkedtrails/waymarked-trails-site#370]
Hi,
this is the place in question:
https://hiking.waymarkedtrails.org/#?map=14!45.9056!15.9838

There are a lot of routes with the red-white circle. They all have a ref, and that's the tag that differentiates them. Right now, that ref is not showing on routes that are correctly tagged with the osmc:symbol. It's not easy to see which paths are which. What's the solution?

One solution was to contact you and see if it's possible to render the ref alongside the symbol. If that's not possible, I will have to resort to tagging the routes with the same osmc symbol, but with a black number over it. That is not entirely correct, because the route itself doesn't have marks with numbers on them, only circles. But it has numbers at start and intersections, so it's not entirely false either. Now that I look at Europe, it seems this is the way to tag osmc:symbols.

What's your opinion?
Anyway, thanks for your project, it's very helpful.

**Comment by mnalis** [comment]
I agree, if there are both `ref` and `osmc:symbol` present (e.g. [here](https://www.openstreetmap.org/relation/8550038)), the ref should take precedence (or, even better, both should be shown - but that is likely more complex to implement)

Related OSM forum discussion: https://community.openstreetmap.org/t/tagiranje-planinarskih-staza-u-hrvatskoj/99704
